src/mono/wasm/debugger/DebuggerTestSuite/Inspector.cs

```diff
@@ -199,6 +199,16 @@ async Task OnMessage(string method, JObject args, CancellationToken token)
             bool fail = false;
             switch (method)
             {
+                case "Target.attachedToTarget":
+                {
+                    var sessionId = new SessionId(args["sessionId"]?.Value<string>());
+                    await Client.SendCommand(sessionId, "Profiler.enable", null, token);
```

Comment:
We should use the code for handling these init commands in `OpenSessionAsync` here also. It could be extracted to a new method. It can be done in a subsequent PR too.